docs/data/regions.yaml

```diff
@@ -62,23 +67,11 @@ europe:
             - "35.246.248.138"
             - "35.246.184.45"
             - "35.242.229.239"
-    fr-1:
-        name: France (fr-1)
-        label: fr-1.platform.sh
-        provider: Orange Cloud
-        outbound_ips: 
-            - "90.84.47.148"
-            - "90.84.46.222"
-            - "90.84.46.40"
-        inbound_location: gw.fr-1.platformsh.site
-        inbound_ips:
-            - "90.84.47.148"
-            - "90.84.46.222"
-            - "90.84.46.40"
     fr-3:
         name: France (fr-3)
         label: fr-3.platform.sh
         provider: OVHcloud
+        available: true
```

Comment:
:bulb:
Shouldn't the carbon intensity be also directly included in the table to help customer choose which region they want to create from / migrate to? 

The carbon intensity of a region is shown during the project creation step, but there's no overview available (only solution is the CLI) which feels cumbersome.

Reply:
I guess that would be possible, though the number could be dynamic. Seems slightly out of scope here.

Reply:
It's indeed out of scope but might be nice to have.